travel.py

```diff
@@ -9,6 +9,44 @@
 sys.setdefaultencoding("utf-8")
 
 
+# 'nick' : ['dep', 'dest', 'time', 'date']
```

Comment:
@Adman this inherently assumes that you are always looking for the next means of transportation of the last searched result. That is probably wrong as I can certainly think of situations where you'd search for someting in one channel and want to type `!bus next` in another one (in a different discussion).

I believe it is safe to ignore this for now but we should keep it in mind in general.